acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/StaticNodesUtils.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.tests.acceptance.dsl;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class StaticNodesUtils {
+
+  public static Path createStaticNodesFile(final Path directory, final List<String> staticNodes) {
+    try {
+      final Path tempFile = Files.createTempFile(directory, "", "");
+      tempFile.toFile().deleteOnExit();
+
+      final Path staticNodesFile = tempFile.getParent().resolve("static-nodes.json");
+      Files.move(tempFile, staticNodesFile);
+      staticNodesFile.toFile().deleteOnExit();
```

Comment:
can we override the file name "static-nodes.json" later on at Pantheon CLI? if we can, then we can avoid the `Files.move`.

Reply:
The other option is to use Files.createTempDirectory(directory, ...) and then create static-nodes.json in it ...

Reply:
The name is hardcoded to `static-nodes.json`. The reason I'm not creating a temp directory is 'cause we already have the node home directory to use (that's where the static-nodes.json) file should be.

I'll take a look and see if I can optimize this in my following PR.